composed-variables/composed-variables.css

```diff
@@ -46,12 +46,14 @@
   --sbb-grid-base-gutter-responsive: var(--sbb-layout-base-grid-gutter-responsive-zero);
 
   /* Button */
-  --sbb-size-button-m-min-height: var(--sbb-size-button-m-min-height-zero);
-  --sbb-size-button-l-min-height: var(--sbb-size-button-l-min-height-zero);
+  --sbb-size-button-s-min-height: var(--sbb-size-element-xs-zero);
```

Comment:
There was probably a misunderstanding: the composed tokens should only contain the new one --sbb-size-element<size>